src/response-model.js

```diff
@@ -0,0 +1,61 @@
+import {
+  getWelcomeRoundQuestion,
+  getRoundQuestions
+} from './questionStorage';
+
+var ResponseModel = class ResponseModel{
+
+   constructor(state) {
+
+    var sessionAttributes = {},
+    speechOutput = "",
+    shouldEndSession = false,
+    rounds = getRoundQuestions(),
+    roundQuestion = getWelcomeRoundQuestion(),
+    correctAnswers = rounds[0].answers,
+    repromptText = getWelcomeRoundQuestion(),
+    CARD_TITLE = "ROUND_ONE";
+
+    if (state === "INTRO"){
+      var speechOutput = "Welcome to Family Feud. Lets begin with the first round. ";
+    }
+
+    speechOutput += repromptText;
+    sessionAttributes = {
+      "speechOutput": speechOutput,
+      "repromptText": repromptText,
+      "roundQuestion": roundQuestion,
+      "correctAnswers": correctAnswers,
+      "score": 0,
+      "currentRound": 0
+    }
+
+    return sessionAttributes;
```

Comment:
when you return something from a constructor, you override the class constructor... this probably isn't what you want to do with your class.